lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -10,10 +10,10 @@
  *
  * @param array $params
  *   An associative array of conditions to filter by. Possible keys:
- *   - nid: (string) A campaign node nid to filter by.
- *   - rbid: (string) A reportback rbid to filter by.
+ *   - nid: (string or array) A campaign node nid to filter by.
+ *   - rbid: (string or array) A reportback rbid to filter by.
  *   - random: (boolean) If set, randomly sort the results.
- *   - runs: (string) Campaign run nid(s) to filter by. 
+ *   - runs: (string or array) Campaign run nid(s) to filter by. 
```

Comment:
I believe these are all strings because we pass a string`12,234,89,23` when we want multiple runs, or campaigns... but then line 70 below looks like we are actually no longer passing an array...